src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4499,42 +4650,45 @@ internal class LocalState : AbstractLocalState
         internal struct LocalState : AbstractLocalState
 #endif
         {
-            // Consider storing nullability rather than non-nullability
-            // or perhaps expose as nullability from `this[int]` even if stored as non-nullability.
-            private BitVector _knownNullState; // No diagnostics should be derived from a variable with a bit set to 0.
-            private BitVector _notNull;
+            private ArrayBuilder<NullableAnnotation> _state;
+            public bool Reachable { get; set; }
```

Comment:
>bool Reachable { get; set; } [](start = 19, length = 28)

`readonly`?

Reply:
> readonly?

Coming in the next commit.


---
In reply to: [231340717](https://github.com/dotnet/roslyn/pull/30913#discussion_r231340717) [](ancestors = 231340717)